enterprise/premium-skins-and-icon-packs/snow-demo.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Premium Skins and Icon Packs demo
+title_nav: Snow Demo
+description: Snow Demo
+keywords: skin skins icon icons customize theme
+---
+
+The Snow skin is designed to as a modern and lightweight editor for web apps. It looks great together with the Thin icon pack.
```

Comment:
SOmething seems to be wrong here: `is designed to as a modern`? I think `to` needs to be removed.

Reply:
Clearly a case of bad editing 😄